### PR TITLE
Update Privacy Pro translations

### PR DESCRIPTION
**Pull request by miasma13**
Task/Issue URL: https://app.asana.com/0/0/1208750072186506/f
CC: @bwaresiak 

**Description**:
Update Privacy Pro and VPN related translations.

**Steps to test this PR**:
Change preferred language for the app and verify completeness for:
- Privacy Pro settings when not signed in
- Privacy Pro settings when signed in
- VPN settings and popover views

**Definition of Done (Internal Only)**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 15
* [ ] iOS 16
* [ ] iOS 17

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/da.lproj/Localizable.strings [hunk 1]
```diff
@@ -1080,25 +1077,31 @@
 "duckPlayer.never.label" = "Aldrig";
 
 /* Body text for the modal feature explanation */
-"duckplayer.presentation.modal.body" = "Duck Player giver dig mulighed for at se YouTube uden målrettede annoncer i en biograflignende oplevelse i DuckDuckGo, og det, du ser, påvirker ikke dine anbefalinger.";
+"duckplayer.presentation.modal.body" = "Duck Player lets you watch YouTube without targeted ads in DuckDuckGo and what you watch won’t influence your recommendations.";
```
Comment (samsymons) [inline DuckDuckGo/da.lproj/Localizable.strings]: This change seems incorrect?

File: DuckDuckGo/da.lproj/Localizable.strings [hunk 2]
```diff
@@ -1080,25 +1077,31 @@
...
 
+/* Two line title (separated by \n) for the feature explanation */
+"duckplayer.presentation.modal.title.youtube" = "Drukner du i annoncer på YouTube? Ikke med Duck Player!";
+
 /* Footer label in the settings screen for Duck Player */
 "duckplayer.settings.footer" = "DuckDuckGo giver dig de Privacy Essentials, du har brug for for at beskytte dig selv, når du surfer på nettet.";
 
 /* Text explaining what Duck Player is in the settings screen. */
-"duckplayer.settings.info-text" = "Duck Player giver en ren seeroplevelse uden målrettede annoncer og forhindrer, at visningsaktivitet påvirker dine YouTube-anbefalinger.";
+"duckplayer.settings.info-text" = "Duck Player lets you watch YouTube without targeted ads in DuckDuckGo and what you watch won’t influence your recommendations.";
```
Comment (samsymons) [inline DuckDuckGo/da.lproj/Localizable.strings]: This one looks incorrect too.

File: DuckDuckGo/da.lproj/Localizable.strings [hunk 3]
```diff
@@ -1529,12 +1589,6 @@
 /* No comment provided by engineer. */
 "keyCommandShowAllTabs" = "Vis alle faner";
 
-/* Please preserve newline character */
-"launchscreenWelcomeMessage" = "Velkommen til\nDuckDuckGo!";
-
-/* No comment provided by engineer. */
-"LOREM IPSUM" = "LOREM IPSUM";
```
Comment (samsymons) [inline DuckDuckGo/da.lproj/Localizable.strings:1536]: 🫡

**Review by samsymons — APPROVED** [review]
Approving, though with a comment about some Duck Player strings, looks like some of these have reverted to English. Otherwise, I understand that the remaining English strings like VPN tips will get translations in a separate PR.

Simulator Screenshot - iPhone 16 Pro Max - 2024-12-09 at 18 07 13

File: DuckDuckGo/da.lproj/Localizable.strings (hunk 1 above)
Comment (miasma13) [reply DuckDuckGo/da.lproj/Localizable.strings]: This was one of two excluded `duckplayer` strings and apparently it also changed the value. Removed exclusion and re-imported the strings, should be fine now.

**Pull request merged**